controllers/update_labels.go

```diff
@@ -80,7 +80,7 @@ func (updateLabels) reconcile(ctx context.Context, r *FoundationDBClusterReconci
 		}
 
 		pvc, ok := pvcMap[processGroup.ProcessGroupID]
-		if !ok || pod == nil {
```

Comment:
That `pod == nil` statement was probably a relict of some refactoring.

Reply:
So "pod" can never be "nil" if "ok == true", right?

Reply:
Pod can be nil but that doesn't matter since the Pod struct is never used in the statements below. The `pod == nil` was a leftover from some previous refactoring.